CMakeLists.txt

```diff
@@ -97,6 +91,13 @@ IF(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     ELSE()
         MESSAGE(WARNING "\nCannot determine the version of the compiler selected to build GNU Radio (GCC : ${CMAKE_CXX_COMPILER}). This build may or not work. We highly recommend using GCC version ${GCC_MIN_VERSION} or more recent.")
     ENDIF()
+    # Release flags: -O3 enables tree vectorization
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O2")
+    set(CMAKE_CXX_FLAGS_RELEASE "-O3 -DNDEBUG")
```

Comment:
This is the default in CMake if I remember correctly, i.e. this line can/should be removed.

Is there any reason why ``${CMAKE_BUILD_TYPE}`` is *not* reused as above?

Reply:
I was a bit surprised to hear that `-O3` is the default for `Release` and it turns out that [it is](https://gitlab.kitware.com/cmake/cmake/blob/v3.16.3/Modules/Compiler/GNU.cmake#L56-59) but some distributions like Fedora [patch it and and change it to `-O2`](https://src.fedoraproject.org/rpms/cmake/blob/master/f/cmake-fedora-flag_release.patch)

Reply:
Yup! Fedora user here.

Reply:
To be clear, this line is the entire reason I'm doing this. My Fedora defaults to `-O2` and I want those juicy optimizations.